content/en/platform/corda/4.7/enterprise/node/operating/node-database-developer.md

```diff
@@ -44,7 +44,7 @@ Setting up a Corda node to connect to a database requires:
 * [Run the node](#db-setup-developer-step-3-ref) to auto-create schema objects
```

Comment:
Header links that need fixing:

- node-database-developer.html#node-database-developer-database-schema-setup-ref
- node-database-developer.html#node-database-developer-database-schema-setup-ref
- node-database-developer.html#node-database-developer-database-schema-setup-when-deploying-a-new-cordapp-ref
- node-database-developer.html#node-database-developer-database-schema-update-when-upgrading-a-new-cordapp-ref
- node-database-developer.html#node-database-developer-database-schema-cleanup-ref
- node-database-developer.html#db-setup-developer-step-1-ref
- node-database-developer.html#db-setup-developer-step-2-ref
- node-database-developer.html#db-setup-developer-step-3-ref
- node-database-admin.html#db-setup-step-3-ref
- node-database-developer.html#db-setup-developer-step-1-ref